app/views/components/radioButtons.scala.html

```diff
@@ -35,7 +35,7 @@
 @govukRadios(
     Radios(
         name = fieldName,
-        attributes = Map("id" -> fieldName),
+        attributes = Map("id" -> s"$fieldName-radios"),
```

Comment:
i think there might be an idprefix property of Radios u can use @CarolineS-QA

Reply:
Oh yea, I just edited this because it was already there. But can update